benchmark_runner/common/ssh/ssh.py

```diff
@@ -22,6 +22,7 @@ def run(self, cmd: str, is_check: bool = False, background: bool = False):
         :return:
         """
         try:
+            print("|", cmd)
```

Comment:
please replace it to logger.info.
Do we really need it ?

Reply:
same as below,

> I'll remove this commit, I tried to limit the flow of error messages / length of callstack to made my debugging easier, but that's personal taste